api4/post_test.go

```diff
@@ -881,6 +881,7 @@ func TestPatchPost(t *testing.T) {
 		rpost, resp = Client.PatchPost(post.Id, patch)
 		CheckNoError(t, resp)
 
+		sort.Strings(fileIds[0:2])
```

Comment:
This is because the `RemoveDuplicateStrings` sorts the strings. Hence the output is sorted. I don't believe the API makes any sort of guarantees that the output will be in a fixed order. So the in-place de-dup should be fine.

> Wondering if we could move it after the initial for loop and sort the whole thing?

Yeah sure. :+1: